sips/sip-018/sip-018-signed-structured-data.md

```diff
@@ -0,0 +1,204 @@
+# Preamble
+
+SIP Number: 018
+
+Title: Signed Structured Data
+
+Author: Marvin Janssen <https://github.com/MarvinJanssen>
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Accepted
+
+Created: 28 December 2021
+
+License: CC0-1.0
+
+Sign-off: Jude Nelson <jude@stacks.org>, Technical Steering Committee Chair
+
+Layer: Applications
+
+Discussions-To: https://github.com/stacksgov/sips
+
+# Abstract
+
+The Signed Structured Data specification describes a standard way to deterministically encode and sign Structured Data. Structured Data is data that can be represented in human-readable format and used in applications and smart contracts. The aim of the standard is to produce signatures that are straightforward and inexpensive to verify by smart contracts.
```

Comment:
Formatting: can you make it so that each line is no more than ~100 characters long?  Otherwise, patching this document will generate a lot of merge conflicts.

Reply:
Can you be more specific about "Structured Data"?  From this document, it looks like "Structured Data" is just any Clarity data that has a canonical encoding according to SIP 005 (so, Clarity values).  The "human readable" descriptor does not appear to be relevant or informative for what you're trying to do here.

Reply:
In light of other things you've said on Discord, I think I now see what you mean here.  You want it to be the case that the data being encoded has a representation that's amenable to human interpretation.  It's not limited to Clarity values, nor does it exclude e.g. a bare hash with a preimage known to the signer.  Is that understanding correct?